### ISSUE TITLE
Cannot use interfaces from @progress/kendo-angular-charts when targeting es2015

### ISSUE THREAD

**Issue by AlexKichkailo** [telerik/kendo-angular#2355]
When trying to use an interface from @progress/kendo-angular-charts, an error is shown during the build if tsconfig targets es2015. There are no errors if tsconfig targets es5.

1) create a new Angular 8 application using ng new ang8.
2) install @progress/kendo-angular-charts
3) create a new file series-wrapper.ts.
`import { Series } from '@progress/kendo-angular-charts';

export class SeriesWrapper {
    constructor(
        public series: Series
    ) { }
}
`
4) Use the file in app.component.ts
public seriesWrapper = new SeriesWrapper(null);
5) Build the project with npm run start

Build fails with the following error

ERROR in ./src/app/series-wrapper.ts 8:12-18
"export 'Series' was not found in '@progress/kendo-angular-charts'

See attached the attached file with the repro
[ang8.zip](https://github.com/telerik/kendo-angular/files/3326507/ang8.zip)

**Expected behavior**
Build does not fail. Application is working.

**Comment by tsvetomir** [comment]
This looks like a regression in `@angular-devkit/build-angular`. Please, try downgrading to v0.800.0:

```diff
diff --git a/package.json b/package.json
--- a/package.json
+++ b/package.json
@@ -32,7 +32,7 @@
     "@progress/kendo-theme-default": "latest"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.800.0",
+    "@angular-devkit/build-angular": "0.800.0",
     "@angular/cli": "~8.0.4",
     "@angular/compiler-cli": "~8.0.1",
     "@angular/language-service": "~8.0.1",
```

There are a few similar bug reports logged in https://github.com/angular/angular:
* https://github.com/angular/angular/issues/31257
* https://github.com/angular/angular/issues/31279

We'll monitor the issue and try to come up with a workaround, if possible.

**Comment by tsvetomir** [comment]
Root issue is https://github.com/angular/angular-cli/issues/14876 — not sure why it's closed though. It is still broken in `@angular-devkit/build-angular@0.801.0-beta.3`.

**Comment by AlexKichkailo** [comment]
@tsvetomir  yeah, thanks for clarifying. Downgrading to "@angular-devkit/build-angular": "0.800.3", fixes the problem. Please feel free to close this issue if needed.

**Comment by tsvetomir** [comment]
Seems to be fixed in v0.800.5 and later — see https://github.com/angular/angular-cli/issues/14876#issuecomment-506523284
